src/msw/window.cpp

```diff
@@ -5860,6 +5915,36 @@ bool wxWindowMSW::HandlePressAndTap(const wxPoint& pt, WXDWORD flags)
 
     return HandleWindowEvent(event);
 }
+
+bool wxWindowMSW::HandleTouch(WXWPARAM wParam, WXLPARAM lParam)
+{
+    const unsigned count = LOWORD(wParam);
+    TOUCHINPUT info[count];
+    if (GestureFuncs::GetTouchInputInfo() ((HTOUCHINPUT)lParam, count, info, sizeof(TOUCHINPUT)))
+    {
+        for(unsigned i = 0; i < count; i++)
+        {
+            wxPoint pos = ScreenToClient(wxPoint(info[i].x / 100, info[i].y / 100));
```

Comment:
A comment explaining these factors of 100 would be nice.

Reply:
https://msdn.microsoft.com/de-de/library/windows/desktop/dd317334(v=vs.85).aspx
This member is indicated in hundredths of a pixel of physical screen coordinates.